code/modules/spacepods/spacepod.dm

```diff
@@ -7,7 +7,7 @@
 
 /obj/item/weapon/pod_paint_bucket
 	name = "space pod paintkit"
-	desc = "Pimp your ride"
+	desc = "Pimp your ride.It is highly recommended the user does not drink his paint and the paint is only used for painting."
```

Comment:
This is very awkwardly worded, awkwardly punctuated, and assumes the user is male. Why not just say:

"Pimp your ride. Do not drink." or something if you really wanna change this.